result/nd.yaml

```diff
@@ -0,0 +1,73 @@
+openapi: 3.1.0
+
+info:
+  title: Neighbor Discovery table models
+  version: ^0.0.0
+
+components:
+  schemas:
+    Nd.Entries.Request:
+      description: >-
+        The request to retrieve Neighbor Discovery entries per IPv6 interface.
```

Comment:
The request to retrieve Neighbor Discovery entries for IPv6 interface(s).

Reply:
Updated.